src/vs/workbench/contrib/extensions/browser/extensionsViewlet.ts

```diff
@@ -452,8 +452,8 @@ export class ExtensionsViewPaneContainer extends ViewPaneContainer implements IE
 				}
 			},
 			onDragOver: (e: DragEvent) => {
-				if (e.dataTransfer) {
-					e.dataTransfer.dropEffect = this.isSupportedDragElement(e) ? 'copy' : 'none';
+				if (this.isSupportedDragElement(e)) {
+					e.dataTransfer!.dropEffect = 'copy';
```

Comment:
Is it safe to assume `e.dataTransfer` is not undefined?

Reply:
`isSupportedDragElement` checks that `e.dataTransfer` is not `undefined`.